core/templates/dev/head/domain/utilities/ImageFileObjectFactory.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Object factory for creating image files.
+ */
+
+oppia.factory('ImageFileObjectFactory', function() {
```

Comment:
Hi @ishucr7! We prefer to use the array syntax for dependency injection. 
The log:
```bash
home/travis/build/oppia/oppia/core/templates/dev/head/domain/utilities/ImageFileObjectFactory.js
  19:1  error  You should use the array syntax for DI  angular/di
✖ 1 problem (1 error, 0 warnings)
```
Thanks!

Reply:
Done!